tests/test_basic_arithmetic.py

```diff
@@ -1,5 +1,11 @@
+# Prevent import errors from optional algorithmic submodule during arithmetic tests
+import sys
+import types
+
 import pytest
 
+sys.modules["reasoning_gym.algorithmic"] = types.ModuleType("reasoning_gym.algorithmic")
```

Comment:
Out of interest why is this necessary here? I'd expect the algorithmic submodule to always be available when running the tests so we shouldn't need to mock it, right?

Reply:
you're correct, it is not necessary, let me remove it